lib/commandlineio.rb

```diff
@@ -0,0 +1,48 @@
+require_relative "blackjack_score"
+
+class CommandlineIO
+
+  def initialize(output = $stdout, input = $stdin)
+    @output = output
+    @input = input
+  end
+
+  def ask_how_many_players
+    @output.puts "1 or 2 player?"
+  end
```

Comment:
if you had an `attr_reader` for `:output` then you would not need to always use `@output`

it's a matter of personal preference which to use.  Ruby's `attr_reader`, `attr_writer` and `attr_accessor` are very useful though and so I'd like for you to learn about them - again might make a nice blog post